packages/core-data/src/reducer.js

```diff
@@ -281,7 +281,6 @@ function entity( entityConfig ) {
 								pending:
 									action.type === 'SAVE_ENTITY_RECORD_START',
 								error: action.error,
-								isAutosave: action.isAutosave,
```

Comment:
This removal will break the `isAutosavingEntityRecord` selector. Which is not currently used by any Gutenberg code, but is a part of public API.